CHANGELOG.md

```diff
@@ -12,6 +12,15 @@ For older changes see the [archived Singularity change log](https://github.com/a
   but `fakeroot-sysv` is much faster.
 - Fix the `--rocm` flag in combination with `-c` / `-C` by forwarding all
   `/dri/render*` devices into the container.
+- Fix the locating of shared libraries when running `unsquashfs` from a
+  non-standard location the way conda does.
+- Fix the creation of missing bind points when using image binding with
+  underlay.
+- Change the error when an overlay image is not writable into a warning
+  that suggests adding `:ro` to make it read only or using `--fakeroot`.
+
+### Bug fixes
+
```

Comment:
Don't add back the Bug fixes here in this branch.

Reply:
removed this section.